pkg/sync/generate_images.go

```diff
@@ -12,6 +12,11 @@ import (
 
 func (o Operation) generateImages(rel release) step {
 	return func() error {
+		// Check if DockerfileGen configuration is provided
+		if o.Config.SkipImage {
```

Comment:
Can't we skip based on the `o.Config.DockerfileGen` not being configured, instead of introducing this flag?

Reply:
True. Thinking...